cli/tests/cli-reference@.md.snap

```diff
@@ -1,6 +1,7 @@
 ---
 source: cli/tests/test_generate_md_cli_help.rs
 description: "AUTO-GENERATED FILE, DO NOT EDIT. This cli reference is generated by a test as an `insta` snapshot. MkDocs includes this snapshot from docs/cli-reference.md."
+snapshot_kind: text
```

Comment:
I had not noticed that this line gets added or removed depending on your `cargo-insta` version. Looking at https://github.com/mitsuhiko/insta/issues/676, they've decided to not include it in the future. I don't mind it being in this version for now. I was just sharing what I found in case others are wondering the.

Reply:
My version:
```
$ cargo insta --version
cargo-insta 1.41.1
```